### DISCUSSION TITLE
what git software tools are FIMS people using?

### DISCUSSION THREAD

**Discussion by iantaylor-NOAA** [NOAA-FIMS/FIMS#457] [Polls]
One of the items in the [Turning ideas into actions](https://docs.google.com/document/d/1eDbt0QU4zYjqzzyejc0tykjMU-pOHGb_2jWpdmwhcRs/edit) discussion was about training and learning more git skills, including how to work better with a large team.

I think that a useful element of this would be learning what tools folks use to work with git. This will help focus future training on the most common tools or perhaps inspire folks to switch to a more common tool if they are an outlier.

**Comment by k-doering-NOAA** [comment]
Thanks @iantaylor-NOAA , I wonder if we want to start some sort of a FIMS user guide doc somewhere that we can link to, like the PEP team has? It seems like it might be helpful to have a reference of who is using what (I'm not it's possible to tell from the poll?

**Reply by kellijohnson-NOAA** [reply]
I have found spreadsheets for things like these are hard to keep up to date. I suggest that we just use the information in the poll to make things incrementally better for the team.

**Comment by kellijohnson-NOAA** [comment]
Question for those using Git Kraken, would you be willing to switch to using git lens within VS Code, which I believe is maintained by the same people that wrote Git Kraken so the user interface is similar?

**Comment by kellijohnson-NOAA** [comment]
I think that the git and GitHub training provided by @e-perl-NOAA, @jonpeake, and others was super helpful for getting people up to speed with git. Those training resources are available [here](https://nmfs-opensci.github.io/GitHub-Clinic/). As well as other great information is available on https://github.com/nmfs-opensci. I am going to close this Discussion for now but feel free to reopen it as questions come up.
